crates/sncast/src/starknet_commands/call.rs

```diff
@@ -11,7 +11,7 @@ use starknet::providers::{JsonRpcClient, Provider};
 #[command(about = "Call a contract instance on Starknet", long_about = None)]
 pub struct Call {
     /// Address of the called contract (hex)
-    #[clap(short = 'a', long)]
+    #[clap(short = 'd', long)]
```

Comment:
Maybe `-c` would make more sense? From `c`ontract

Reply:
If you refer to `Call` only - it's possible, but I think it's better to maintain consistency and have short option 
 of `contract_address` the same for every command. Then it's not possible as  `c` is already used by `calldata` in `Invoke`.